drivers/serial/Kconfig.rpi_pico

```diff
@@ -9,3 +9,4 @@ config UART_RPI_PICO
 	default $(dt_compat_enabled,$(DT_COMPAT_RPI_PICO_UART))
 	select SERIAL_HAS_DRIVER
 	select PICOSDK_USE_UART
+	select SERIAL_SUPPORT_INTERRUPT
```

Comment:
this is up to the board to decide, I guess

Reply:
In all drivers that I saw in zephyr this was here.

Reply:
sorry, got confused with `CONFIG_UART_INTERRUPT_DRIVEN`, my fault.